sdk/confidentialledger/Azure.Security.CodeTransparency/tests/CodeTransparencyClientLiveTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.Security.CodeTransparency.Tests
+{
+    public class CodeTransparencyClientLiveTests: ClientTestBase
+    {
+        private readonly string _canaryUrl = "https://cts-canary.confidential-ledger.azure.com";
+        private readonly string _canaryIdentityServiceUrl = "https://identity.confidential-ledger.core.azure.com";
```

Comment:
```suggestion
```

These cannot be used for live testing nor should they be committed to the repository.  The [Test Resources](https://github.com/Azure/azure-sdk-for-net/blob/main/eng/common/TestResources/README.md) system must be used for both local and pipeline live test runs.

Reply:
We do not have ARM support yet, will remove the live test